### DISCUSSION TITLE
Problem using "into" operator to duplicate channels

### DISCUSSION THREAD

**Discussion by espelpz** [nextflow-io/nextflow#2694] [Q&A]
I need to use the same channel multiple times so I'm trying to create copies of it using the into operator in this way:

Saving path of interest in a channel:

`if (params.fasta) { ch_fasta = file(params.fasta, checkIfExists: true) }`

Replicating that channel for using its content in different nextflow processes:

`ch_fasta
    .into { ch_fasta_bwamap
            ch_fasta_bwaindex
            ch_fasta_picarddict
            ch_fasta_bwamapcons
            ch_fasta_filtercons
            ch_fasta_clipbam
            ch_fasta_mutect2
            ch_fasta_vardict }
`

However, when I try to do this I obtain this error message from nextflow:

`mar-03 15:10:45.170 [main] DEBUG nextflow.Session - Session aborted -- Cause: No signature of method: sun.nio.fs.UnixPath.into() is applicable for argument types: (Script_2a703cbc$_runScript_closure2) values: [Script_2a703cbc$_runScript_closure2@2dc3ece8]
Possible solutions: inits(), init(), any(), min(), list(), find()
mar-03 15:10:45.193 [main] ERROR nextflow.cli.Launcher - @unknown
groovy.lang.MissingMethodException: No signature of method: sun.nio.fs.UnixPath.into() is applicable for argument types: (Script_2a703cbc$_runScript_closure2) values: [Script_2a703cbc$_runScript_closure2@2dc3ece8]
Possible solutions: inits(), init(), any(), min(), list(), find()
`

So, I'm not able to duplicate the channel eventhough in the past I have used this operator in this way to do that... I don't know what I'm doing wrong or how I can do this in another way.

Hope someone can help me, thank you!

**Comment by bentsherman** [comment]
You didn't actually create a channel at the start, just a file object. You should instead use `Channel.fromPath()`.
